roles/oVirt.datacenter-cleanup/tasks/find_hosted_engine_details.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Find existing VMs
+  ovirt_vms_facts:
+    auth: "{{ ovirt_auth }}"
+    pattern: "datacenter={{ data_center_name }}"
+
+- name: Find HE info if HE
+  set_fact:
+    hosted_engine: "{{ ovirt_vms | selectattr(search_key, 'equalto', search_val) | list | count > 0 }}"
+  vars:
+    search_key: origin
+    search_val: managed_hosted_engine
```

Comment:
Should we support also: https://github.com/oVirt/ovirt-engine/blob/06ce94db7eda2567eeb5471d899f9df901f12f15/backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/OriginType.java#L15 ?